src/Model/Package.php

```diff
@@ -14,12 +14,6 @@ class Package extends DataObject
         'Type' => 'Varchar(255)',
     ];
 
-    private static $summary_fields = [
```

Comment:
By default all fields are summary fields, as I recall. I don't agree that `Type` `silverstripe-module` is relevant information for export (or plain text summaries anywhere else). Could you please revert this change?

Reply:
Can you explain again why it's needed? When I checked the summary fields were not being used to generate the report

Reply:
There's a bug here. I'm going to fix it.

Reply:
My review comment summarises the notion too (in the conversation tab, not a diff comment like this thread), if that provides more context to my meaning :)

Reply:
I've pushed a commit which ensures that the same columns are used for exporting to CSV, which removes the confusion around this piece of code

Reply:
I still disagree with this approach. I think using the inbuilt functionality provided by SilverStripe for this purpose (`summary_fields` config) is more consistent overall and causes less cognitive friction when reading.

If anything I'd prefer that `summary_fields` here was used in the generation of the columns, having the modification for the HTML output happen where that's relevant - in the columns function on the report. The default output for a CSV export button is to use the `summary_fields` of the model, so that logic in `getReportField` is no longer needed and would actually result in both a reduction of code and confusion in this class - following the idiom 'simplest is best', which I think is what we're really striving for here.

Please see the other two comments.